ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts

```diff
@@ -0,0 +1,70 @@
+import { Model, NodeModel } from '@patternfly/react-topology';
+
+import { ListenPort } from 'types/networkFlow.proto';
+
+type Override<T1, T2> = Omit<T1, keyof T2> & T2;
```

Comment:
TIL

Reply:
I was thinking this `Override` type might be useful to have somewhere more global so it can be used in other instances. What do you think? (cc: @vjwilson @alwayshooin @pedrottimark @dvail @bradr5)

Reply:
+1

I've been putting things like this in `src/utils/type.utils.ts` as they come up.